department-of-veterans-affairs/abd-vro
# MCP-2137 Add sufficientEvidence flag to assessment results.

MCP-2137 Add sufficientEvidence flag to assessment results.

## What was the problem?
We need to add a sufficient evidence flag to claims after MAS automated claim processes the claims.

Associated tickets or Slack threads:

- [MCP-2137](https://amida.atlassian.net/browse/MCP-2137)

## How does this fix it?[^1]


## How to test this PR
- Step 1 Run MAS automated claim endpoint
- Step 2 Allow the MAS endpoint to do its thing, it takes about a minute.
- Step 3 Check the assessment result in the VRO database for the correct flag, [] = false and [v] = true.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
